backend/engine/plugins/checkov/main.py

```diff
@@ -21,29 +34,60 @@ def main():
     LOG.info("Executing Checkov")
     args = utils.parse_args()
     path = abspath(args.path)
-    output = run_checkov(path, args.config)
+
+    errors: list[str] = []
+
+    (temp_vol_name, temp_vol_mount) = str(args.engine_vars.get("temp_vol_name", "")).split(":")
+    if not temp_vol_name or not temp_vol_mount:
+        errors.append("Temporary volume not provided")
+
+    (working_src, working_mount) = str(args.engine_vars.get("working_mount", "")).split(":")
+    if not working_src or not working_mount:
+        errors.append("Working volume not provided")
```

Comment:
How are `temp_vol_name` and `working_mount` defined? Do we control them?

Reply:
These are set here: https://github.com/WarnerMedia/artemis/blob/zoogiezork/checkov-docker-2/backend/engine/utils/plugin.py#L167-L168